bedrock/settings/__init__.py

```diff
@@ -38,13 +39,13 @@
     csp.constants.SELF,
     # NOTE: Check if these need to be in the `_csp_form_action` list as well since we often
     # progressively enhance forms by using Javascript.
-    "cjms.services.mozilla.com",
     "o1069899.ingest.sentry.io",
     "o1069899.sentry.io",
     "region1.google-analytics.com",
-    "stage.cjms.nonprod.cloudops.mozgcp.net",
     "www.google-analytics.com",
     "www.googletagmanager.com",
+    # Just get the CJMS domain from the URL, nothing else.
+    urlparse(CJMS_AFFILIATE_ENDPOINT).netloc,
```

Comment:
I think it'd be safe to skip importing urlparse and just use the constant incl. its protocol and slash for the same effect… but I mostly wanted to mention the removal of CJMS in [#16269/files](https://github.com/mozilla/bedrock/pull/16269/files#diff-eaca14ba79bd8caf620f2740e240a03b6e2d192e1df89dc259ec63894da1f62d) (whichever lands first) so this will be short-lived one way or another…

Reply:
Yeah I'm hoping that PR gets merged first. :) But I liked this to simply strip it down to the base URL, which is what I think we'd do if it were hard-coded as a string here. Thanks for looking at this.

Reply:
The PR was merged and this is now rebased without the CJMS lines.